src/Uno.UI/UI/Xaml/UIElement.cs

```diff
@@ -107,6 +107,8 @@ internal virtual bool GetDefaultValue2(DependencyProperty property, out object d
 
 		internal Size AssignedActualSize { get; set; }
 
+		internal bool IsLeavingFrame { get; set; }
```

Comment:
Validate if you can use a flag instead, see with @carldebilly for compatibility.

Reply:
@carldebilly can I merge this for now or should I wait for your changes first?

Reply:
@carldebilly @jeromelaban seems I cannot easily use the facilities from Carl's PR, for two reasons:

- The `LayoutFlags` are netstd specific, but this property needs to apply to all targets
- It is rather a focus-related thing, not layout

I would say it would be easiest to merge this for now and maybe create some other flags enum for non-layout bools?

Reply:
Let's merge as is for now.